.build/automation/stages/validate.yml

```diff
@@ -12,6 +12,19 @@ stages:
             clean: true
             persistCredentials: true
 
+          - task: CmdLine@1
```

Comment:
Why is this required? You are essentially setting the value back to itself?

Reply:
You are asking about the Set Token step? I'm not sure why (or if) it's needed, but we seem to do everywhere before calling the provisionator. I copied it from the Windows section above. It's also in Hot Reload here: https://github.com/xamarin/UITools/blob/a90b49522b5aadc38c8f63992cf80c1e63d60e21/HotReload/build/automation/stages/validate.yml#L189
Do think it doesn't actually do anything? It seems like it's setting the GITHUB_TOKEN environment variable from the pipeline property.

Reply:
@mcumming ^^^

Reply:
Indeed, this wasn't needed. I removed it, from both Mac and Windows.